### PR TITLE
Add a FAQ entry about "Why Mixer?"

### PR DESCRIPTION
**Pull request by geeknoid**
- Also a few fixes in other FAQ entries.

- Force the right-hand scrollbar on, which eliminates the very annoying left/right jiggle that
was happening whenever loading a new page on istio.io.

Staging: https://geeknoid.github.io/istio.github.io/faq/

### CODE REVIEW

File: _faq/why-mixer.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Why does Istio need Mixer?
+order: 135
+type: markdown
+---
+{% include home.html %}
+
+Mixer provides a rich intermediation layer between the Istio components as well as Istio-based services,
```
Comment (douglas-reid) [inline _faq/why-mixer.md]: i might add a comma between 'components' and 'as' here

File: _faq/why-mixer.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+layer enables operators to have rich insights and control over service behavior without requiring
+changes to service binaries.
+
+Mixer is designed as a stand-alone component, distinct from Envoy, in order to be able
+to manage scaling and reliability concerns independently . The proxy is focused on
+high-performance routing, while Mixer takes responsibility for policy management and telemetry collection.
+Mixer is invoked by Envoy, and in the future will also be invoked directly by Istio-based services.
+
+Envoy implements sophisticated caching, batching, and prefetching, to largely mitigate the
+latency impact of needing to interact with Mixer on the request path.
```
Comment (douglas-reid) [inline _faq/why-mixer.md]: do we want to mention extensibility?

and/or lifecycle concerns (ability to rollout policy controls distinct from proxy rollouts)?  we mention 'scaling and reliability concerns independently', but there are other advantages too, right?

File: _faq/why-mixer.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+type: markdown
+---
+{% include home.html %}
+
+Mixer provides a rich intermediation layer between the Istio components as well as Istio-based services,
+and the infrastructure backends used to perform access control checks and telemetry capture. This
+layer enables operators to have rich insights and control over service behavior without requiring
+changes to service binaries.
+
+Mixer is designed as a stand-alone component, distinct from Envoy, in order to be able
```
Comment (mandarjog) [inline _faq/why-mixer.md]: We should be explicit about "Why is mixer not implemented as an envoy filter" 

* Proxy stability 
  * Complex logic moved to the mixer. Simpler and more reliable proxy.
  * If mixer goes down, proxy can fail open
* Proxy replaceability
  * Proxy can be swapped by using mixer client
  * Can use multiple types of proxies; nginx, envoy all in the same deployment.
* Mixer can run in a separate trust domain
 * Isolates back ends from the proxy
* Services can call the control plane directly. Solves N x M integration problem.
* Proxies are simpler and smaller
 * They can be numerous.
 * Cheaper to run
* Proxy and mixer can scale at different rates

**Comment by mandarjog** [comment]
@geeknoid can you address our comments?

**Comment by geeknoid** [comment]
Yes, I'll address the comments. I misused git and checked in my master branch while I didn't intend to. I'll submit another PR to address the issues shortly.

**Pull request merged**